src/Whoops/Util/TemplateHelper.php

```diff
@@ -72,6 +72,39 @@ public function escapeButPreserveUris($raw)
         );
     }
 
+    /**
+     * Makes sure that the given string breaks on the delimiter.
+     *
+     * @param  string $delimiter
+     * @param  string $s
+     * @return string
+     */
+    public function breakOnDelimiter($delimiter, $s)
```

Comment:
You can replace the implementation of this function with a `str_replace($s, $delimiter, $delimiter . '&shy;')`.
The DOM is simpler, the PHP is simpler.

- [ ] Replace `div.delimiter` with `&shy`s.